src/httpserver/qabstracthttpserver.cpp

```diff
@@ -127,8 +129,12 @@ void QAbstractHttpServerPrivate::handleReadyRead(QTcpSocket *socket,
         Q_EMIT q->missingHandler(*request, socket);
 }
 
+QIODevice* QAbstractHttpServer::createBodyDevice(const QHttpServerRequest& /*request*/) {
```

Comment:
shared pointer if possible

Reply:
I would prefer use the Qt's onwership memory management @lmussier An advice ?

Reply:
Agree with sylvain, we are in Qt world.